src/setup.jl

```diff
@@ -194,7 +194,7 @@ function __init__()
         # R gui eventloop
         isinteractive() && rgui_init()
         # R REPL mode
-        isdefined(Base, :active_repl) && isinteractive() && repl_init(Base.active_repl)
+        isdefined(Base, :active_repl) && isinteractive() && (typeof(Base.active_repl)!=Base.REPL.BasicREPL) && repl_init(Base.active_repl)
```

Comment:
Pedantic stylistic comment: Please remove the parentheses from around the expression (it parses as desired without them) and surround `!=` with spaces.